### PR TITLE
[ISSUE #2128] Fix unit tests occasionally fail to execute.

### PR DESCRIPTION
**Pull request by kevinBobo**
…un test fail"(#2128)

// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-loadbalancer/src/test/java/org/apache/shenyu/loadbalancer/cache/UpstreamCheckTaskTest.java [hunk 1]
```diff
@@ -75,15 +75,15 @@ public void testRun() {
         healthCheckTask.schedule();
         // Wait for the upstream-health-check thread to start.
         Awaitility.await().pollDelay(3, TimeUnit.SECONDS).untilAsserted(() -> assertFalse(healthCheckTask.getCheckStarted().get()));
-        assertTrue(healthCheckTask.getHealthyUpstream().containsKey(selectorId1));
+        assertTrue(healthCheckTask.getUnhealthyUpstream().get(selectorId1).size() > 0);
         // Let it coverage line 151~163
         when(upstream.isHealthy()).thenReturn(false).thenReturn(true);
```
Comment (dengliming) [inline shenyu-loadbalancer/src/test/java/org/apache/shenyu/loadbalancer/cache/UpstreamCheckTaskTest.java:80]: `when(upstream.isHealthy()).thenReturn(false).thenReturn(true);` what does this mean?

**Review by dengliming — CHANGES_REQUESTED** [review]

File: shenyu-loadbalancer/src/test/java/org/apache/shenyu/loadbalancer/cache/UpstreamCheckTaskTest.java (hunk 1 above)
Comment (yu199195) [reply shenyu-loadbalancer/src/test/java/org/apache/shenyu/loadbalancer/cache/UpstreamCheckTaskTest.java:80]: maybe is right

Comment (kevinBobo) [reply shenyu-loadbalancer/src/test/java/org/apache/shenyu/loadbalancer/cache/UpstreamCheckTaskTest.java:80]: I think the original author wants to change to return true when returning false。

**Review by yu199195 — APPROVED** [review]

**Review by dengliming — APPROVED** [review]

File: shenyu-loadbalancer/src/test/java/org/apache/shenyu/loadbalancer/cache/UpstreamCheckTaskTest.java (hunk 1 above)
Comment (dengliming) [reply shenyu-loadbalancer/src/test/java/org/apache/shenyu/loadbalancer/cache/UpstreamCheckTaskTest.java:80]: > I think the original author wants to change to return true when returning false。

We can just keep the last return.

**Pull request merged**